.gitignore

```diff
@@ -1,7 +1,6 @@
 pkg/
 Gemfile.lock
 Gemfile.local
-vendor/
```

Comment:
My team had trouble with vendor/ being in .gitignore and trying to use /build/vendor/custom for our custom modules. I had to add the following line under vendor to our .gitignore file or it wouldn't push our custom modules to the remote repo.
...
vendor/
!/build/vendor
...